### PR TITLE
feat(component): adding "padding" property to customize padding style

### PR DESCRIPTION
**Pull request by pvcresin**
I want an interface to customize the padding.

### CODE REVIEW

**Comment by JimmyCN1** [comment]
Let's get this merged hey :)

**Comment by aronhelser** [comment]
Does this change the default padding? Where was "8px 21px" used before?

**Comment by pvcresin** [comment]
@aronhelser here: src/index.scss

**Comment by pvcresin** [comment]
Conflict resolved

File: src/index.scss [hunk 1]
```diff
@@ -4,7 +4,6 @@
   font-size: 13px;
   left: -999em;
   opacity: 0;
-  padding: 8px 21px;
```
Comment (pvcresin) [inline src/index.scss:7]: current default padding

**Comment by pvcresin** [comment]
CI passed

**Comment by pvcresin** [comment]
Is there anybody here?

**Comment by grallc** [comment]
Hey, any news on this ?

**Comment by aronhelser** [comment]
We really need a maintainer who is actively working with react....

**Comment by cyruscuenca** [comment]
I too, would like padding prop ASAP 😅

**Pull request merged**